### PR TITLE
Fix judge not checking if connected component is maximal

### PR DESCRIPTION
**Pull request by jonremy**

### CODE REVIEW

**Review by lypnol — APPROVED** [review]
👍

**Review by cdancette — APPROVED** [review]

**Comment by cdancette** [comment]
LGTM, :shipit:

**Comment by cdancette** [comment]
:sheep:  it

File: problem-01/judge.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def is_complete_path(graph, path, start):
 
         # Vérifier la connexité
         if not is_complete_path(graph, path, start):
-            return True
+            return False
```
Comment (david-ds) [inline problem-01/judge.py:45]:
```python
return is_complete_graph(graph, path, start)
```
pour chipoter 😘

**Pull request merged**